### PR TITLE
DEVELOPER-1172 Create JBoss Champions Microsite

### PR DESCRIPTION
**Pull request by paulrobinson**

### CODE REVIEW

**Comment by paulrobinson** [comment]
retest this please

File: champions.html.slim [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    .large-12.column
+      img(src=cdn(site.base_url + "/images/champions/roadtoawesome_graphic.png"))
+    .large-12.column
+      img(src=cdn(site.base_url + "/images/champions/jbosschampion_logo.png"))
+      p JBoss Champions is a selected group of community members who are passionate advocate of JBoss technologies under a program sponsored by Red Hat. JBoss Champions use a variety of JBoss technologies and actively share their deep technical expertise about it in the community, within their company and their customers or partners. This could be done in forums, blogs, screencasts, tweets, conferences, social media, whitepapers, articles, books, and other means.
+
+.wide.microsite-text-points
+  .row
+    .large-12.column
+      h2 Who is a JBoss Champion?
```
Comment (pmuir) [inline champions.html.slim]: might be better to use asciidoc here?

File: champions.html.slim [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      p JBoss Champions areas of contributions include, but not limited to:
+      ul
+        li Senior developers, architects, consultants, academia who are using and promoting JBoss technologies
```
Comment (pmuir) [inline champions.html.slim]: The line wrap on the list items is not indenting properly.

File: champions.html.slim (hunk 1 above)
Comment (paulrobinson) [inline champions.html.slim]: Yeah, I thought that too.

File: champions.html.slim [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+hero_class: champions
+---
+
+= define 'hero'
+  .row
+    .large-12.column
+      img(src=cdn(site.base_url + "/images/champions/roadtoawesome_graphic.png"))
+    .large-12.column
+      img(src=cdn(site.base_url + "/images/champions/jbosschampion_logo.png"))
+      p JBoss Champions is a selected group of community members who are passionate advocate of JBoss technologies under a program sponsored by Red Hat. JBoss Champions use a variety of JBoss technologies and actively share their deep technical expertise about it in the community, within their company and their customers or partners. This could be done in forums, blogs, screencasts, tweets, conferences, social media, whitepapers, articles, books, and other means.
```
Comment (pmuir) [inline champions.html.slim]: This overflowing the banner area on my Chrome - see http://screencast.com/t/YErCc6324

File: stylesheets/_champions.scss [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      border-bottom: 0;
+      font-size: 24px;
+      &:hover {
+        border-bottom: 0;
+        color:white;
+        background:$dark-blue;
+      }
+    }
+  }
+}
```
Comment (paulrobinson) [inline stylesheets/_champions.scss]: needs a new-line

File: champions.html.slim (hunk 2 above)
Comment (paulrobinson) [inline champions.html.slim]: @adelasofia please can you take a look at this? 

Look at: https://www.dropbox.com/s/q37hc4qzn42ub3k/Screenshot%202015-02-09%2011.36.20.png?dl=0. Notice that the word "promoting" is not horizontally aligned with the word "Senior".

I tried this, but couldn't get it to work: http://stackoverflow.com/questions/11556493/2nd-line-in-li-start-under-the-bullet-after-css-reset

File: champions.html.slim (hunk 3 above)
Comment (paulrobinson) [inline champions.html.slim]: This seems to happen very occasionally for me too. @adelasofia do you know what's happening here?

**Comment by paulrobinson** [comment]
Closing this PR as it's pretty broken :-( 

Replacing with a new one.

**Pull request closed without merging**